### PR TITLE
fix(editableDropdown): update border-radius inside customRenderer story

### PR DESCRIPTION
**Pull request by saty2103**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/molecules/editableDropdown/__stories__/style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.customTrigger {
+  border-radius: 4px;
```
Comment (anuradha9712) [inline core/components/molecules/editableDropdown/__stories__/style.css]: Please use `var(--spacing-m)` token

File: core/components/molecules/editableDropdown/__stories__/customRenderer.story.jsx [hunk 2]
```diff
@@ -88,7 +96,7 @@ const customCode = `() => {
 
   const customTrigger = (label) => {
     return (
-      <div className="d-flex align-items-center justify-content-between pl-5 pr-4 bg-secondary cursor-pointer w-100">
+      <div className="d-flex customTrigger align-items-center justify-content-between pl-5 pr-4 bg-secondary cursor-pointer w-100">
```
Comment (anuradha9712) [inline core/components/molecules/editableDropdown/__stories__/customRenderer.story.jsx]: Please follow BEM convention for className

Comment (saty2103) [reply core/components/molecules/editableDropdown/__stories__/customRenderer.story.jsx]: Done. Please verify.

File: core/components/molecules/editableDropdown/__stories__/style.css (hunk 1 above)
Comment (saty2103) [reply core/components/molecules/editableDropdown/__stories__/style.css]: Done.

**Review by anuradha9712 — APPROVED** [review]

**Pull request merged**